src/stronghold.hs

```diff
@@ -237,13 +237,14 @@ site runStoreOp =
       hier <- Trees.lastHierarchy hist
       snd <$> Trees.materializedView path hier
 
-    let etag = (Base16.encode . hash . toStrict . Aeson.encode) json
+    let jsonBS = (toStrict . Aeson.encode) json
+    let etag = (Base16.encode . hash) jsonBS
```

Comment:
If you want to be more consistent with the style of the rest of the codebase, then

```
let jsonBS = toStrict $ Aeson.encode json
let etag = Base16.encode $ hash jsonBS
```